grtext/grfont.cpp

```diff
@@ -959,8 +959,8 @@ void grfont_XlateColorChar(int bmp_handle, int x, int y, int index, tFontFileInf
   sptr = (ushort *)ft->char_data[index];
   rowsize_w = bm_rowsize(bmp_handle, 0) / 2;
 
-  // height = SET_MIN(h, ddsfObj.h);
-  // width = SET_MIN(w, ddsfObj.w);
+  // height = std::min(h, ddsfObj.h);
+  // width = std::min(w, ddsfObj.w);
```

Comment:
?

Reply:
it's best to fix commented out code too otherwise it loses all meaning because the macro no longer exists.

Reply:
Sorry but I disagree. The current base is a blend of old and new standards, so having the old macros in comments can be helpful for reference.

Reply:
> You would need to find the old version of the source in order to understand the purpose of the commented out code because the macro no longer exists. Is this the desired outcome?

We have that though. It's the `d3-historical` branch.

Reply:
@JeodC what is gained by forcing someone to look up an old macro?

Reply:
> @JeodC what is gained by forcing someone to look up an old macro?

My comment is no longer valid here.